tests/FSharp.Test.Utilities/ILChecker.fs

```diff
@@ -15,8 +15,7 @@ module ILChecker =
 
     let private exec exe args =
         let arguments = args |> String.concat " "
-        let timeout = 30000
```

Comment:
Running in parallel in CI makes the timings less predictable. There is a `--blame-hang-timeout` to catch actually hanging tests.